src/layer/Popup.js

```diff
@@ -97,6 +97,10 @@ export var Popup = DivOverlay.extend({
 		// Controls the presence of a close button in the popup.
 		closeButton: true,
 
+		// @option closeButtonLabel: String = 'Close popup'
+		// Specifies the 'aria-label' attribute of the close button.

```

Comment:
Let's use backticks here for the attribute name, it will get special formatting in the docs.
```suggestion
		// Specifies the `aria-label` attribute of the close button.

```